example/mistralai_client_dart_example.dart

```diff
@@ -8,15 +8,21 @@ void main() async {
   final client = MistralAIClient(apiKey: mistralApiKey);
 
   // list models
-  final modelsResult = await client.listModels();
-  final models = modelsResult.data?.map((e) => e.id).toList();
+  final modelsResult = await client.listMistralModels();
+  final models = modelsResult?.map((e) {
+    if (e.baseModel != null) {
+      return e.baseModel?.id;
+    } else {
+      return e.fineTunedModel?.id;
+    }
+  }).toList();
   print(models?.join(', '));
 
   // chat without streaming
   const request = ChatCompletionRequest(
     model: 'mistral-small-latest',
     messages: [
-      UserMessage(content: UserMessageContent.string('Hello chat!')),
+      UserMessage(content: 'Hello chat!'),
```

Comment:
In [ba6b353](https://github.com/nomtek/mistralai_client_dart/pull/102/commits/ba6b3530104542ae8c91935fae1cde8198e53cd1) I changed the UserMessage implementation so that it does not introduce breaking changes any more.